data/members/gholad.yml

```diff
@@ -0,0 +1,6 @@
+id: gholad
+display-name: Gholad
+bio: I have interests in functional programming but please calm down, i'm really bad.
+location: France, Paris
+main_link:
+  url: https://hakimba.github.io/oxywa/
```

Comment:
Since your website is in french, I suggest to add the `lang` field in the `main-link` section, You can checkout xvw.yml for some examples!

Reply:
done